tests/test_parsers/test_symbol_scraper_parser.py

```diff
@@ -85,32 +85,38 @@ def setUpClass(cls):
+                {'text': 'The', 'box': [0.24914161764705883, 0.14273611111111115, 0.045513914379084974, 0.014693952020202017, 0]},
+                {'text': 'Smoothed', 'box': [0.3031788235294118, 0.14273611111111115, 0.11873560294117647, 0.015867739267676734, 0]},
+                {'text': 'Possibility', 'box': [0.4300158169934641, 0.14273611111111115, 0.12191432352941173, 0.018932647727272683, 0]}
             ],
             1: [
-                {'text': 'Curves', 'bbox': [0.365513545751634, 0.2131515151515152, 0.04706188398692812, 0.011132455808080727]},
-                {'text': 'Flow', 'bbox': [0.4171823202614379, 0.2131515151515152, 0.027787840359477123, 0.008692116161616165]},
-                {'text': 'RPI,', 'bbox': [0.44990264705882355, 0.2131515151515152, 0.03123896781045743, 0.008503431060606054]}
+                {'text': 'Curves', 'box': [0.365513545751634, 0.2131515151515152, 0.04706188398692812, 0.011132455808080727, 0]},
+                {'text': 'Flow', 'box': [0.4171823202614379, 0.2131515151515152, 0.027787840359477123, 0.008692116161616165, 0]},
+                {'text': 'RPI,', 'box': [0.44990264705882355, 0.2131515151515152, 0.03123896781045743, 0.008503431060606054, 0]}
             ]}
         }
         cls.doc_json = {
-            'text': 'The Smoothed Possibility\nCurves Flow RPI,\n',
-            'page': [{'start': 0, 'end': 41, 'id': 0, 'bbox': [0.24914161764705883, 0.14273611111111115, 0.302788522875817, 0.08154785984848478]}],
-            'token': [{'start': 0, 'end': 3, 'id': 0, 'bbox': [0.24914161764705883, 0.14273611111111115, 0.045513914379084974, 0.014693952020202017]},
-                      {'start': 4, 'end': 12, 'id': 1, 'bbox': [0.3031788235294118, 0.14273611111111115, 0.11873560294117647, 0.015867739267676734]},
-                      {'start': 13, 'end': 24, 'id': 2, 'bbox': [0.4300158169934641, 0.14273611111111115, 0.12191432352941173, 0.018932647727272683]},
-                      {'start': 25, 'end': 31, 'id': 3, 'bbox': [0.365513545751634, 0.2131515151515152, 0.04706188398692812, 0.011132455808080727]},
-                      {'start': 32, 'end': 36, 'id': 4, 'bbox': [0.4171823202614379, 0.2131515151515152, 0.027787840359477123, 0.008692116161616165]},
-                      {'start': 37, 'end': 41, 'id': 5, 'bbox': [0.44990264705882355, 0.2131515151515152, 0.03123896781045743, 0.008503431060606054]}],
-            'row': [{'start': 0, 'end': 24, 'id': 0, 'bbox': [0.24914161764705883, 0.14273611111111115, 0.302788522875817, 0.018932647727272683]},
-                    {'start': 25, 'end': 41, 'id': 1, 'bbox': [0.365513545751634, 0.2131515151515152, 0.115628069117647, 0.011132455808080727]}],
-            'sent': [],
-            'block': []
+            Symbols: 'The Smoothed Possibility\nCurves Flow RPI,\n',
+            Pages: [
+                {'_type': 'SpanGroup',
```

Comment:
BTW, do you like we store the `_type` in the JSON?

Reply:
not really; i think it takes up a lot of space for not much benefit -- it's very obvious whether it's a SPanGroup vs BoxGroup.  SpanGroup contains `spans` key.  i'll remove